app-server/app/src/main/scala/user/User.scala

```diff
@@ -72,6 +73,9 @@ object UserService {
       Query[Users,Users#TableElementType,Seq] = {
     import database.driver.api._
 
+    val sql = query.result.statements.head
+    log.debug(s"Returning users -- SQL: $sql")
```

Comment:
Minor performance enhancement when not running at loglevel `debug`:
1. remove `val sql = ...`
2. `log.debug(s"Returning users -- SQL: ${query.result.statements.head}")